lib/cocina_display/concerns/forms.rb

```diff
@@ -47,6 +47,13 @@ def genres_search
         end.uniq
       end
 
+      # Map-related data to be rendered for display.
+      # Includes map scale, projection info, and geographic coordinates.
+      # @return [Array<DisplayData>]
+      def map_display_data
+        [DisplayData.new(label: I18n.t("cocina_display.field_label.map_data"), values: map_forms + coordinates)]
```

Comment:
Also shouldn't this allow for displayLabel from the cocina?

Reply:
Maybe? https://github.com/sul-dlss/mods_display/blob/5583ac5a353ec32316871c4ceb27712f5bd246d1/lib/mods_display/fields/cartographics.rb#L19